traffic_ops/app/db/migrations/2021081800000000_roles_and_permissions.down.sql

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
> 2021081800000000 with zeros

Why not use `db/admin` to create the migration?

Reply:
it does that?

Reply:
Yep

```shell
go run ./db create_migration my_migration
```

Reply:
neat